core/src/main/scala/org/apache/spark/h2o/H2OConf.scala

```diff
@@ -33,6 +34,8 @@ class H2OConf(val sparkConf: SparkConf) extends Logging with InternalBackendConf
 
   H2OConf.checkDeprecatedOptions(sparkConf)
 
+  protected var nonJVMClientCreds: Option[FlowCredentials] = None
```

Comment:
Note: Doesn't this variable get serialized and sent to each node in internal backend during bootstrap?

Reply:
Good catch, I think we can make it @transient, what do you think?

Reply:
Sounds good to me. I think it would great if we could  to move the variable somewhere else to avoid usage of reflection. Unfortunately, I can't think of a different place.